Makefile

```diff
@@ -70,6 +70,8 @@ test: ## Tests that biomage cmd & subcommand are available
 
 	biomage experiment --help > /dev/null
 	biomage experiment download --help > /dev/null
+	biomage experiment upload --help > /dev/null
```

Comment:
What is this for? you didn't add an upload.py file

Reply:
the upload.py is already there, but not added to the test. I added it here while adding test for the info command.